br-ext/package/optee_client_ext/Config.in

```diff
@@ -8,3 +8,9 @@ config BR2_PACKAGE_OPTEE_CLIENT_EXT_SITE
 	default ""
 	help
 	  The path to the source directory of OP-TEE client
+
+config BR2_PACKAGE_OPTEE_CLIENT_EXT_HW_RPMB
+	bool "Enable real RPMB device"
+	help
+	  Enable real RPMB device support by disabling the RPMB emulation in
+	  tee-supplicant.
```

Comment:
How about `BR2_PACKAGE_OPTEE_CLIENT_EXT_RPMB_EMU` default `y`?

Reply:
+1. Would be more consistent. I'll update. thanks.